aspnetcore/migration/proper-to-2x/index.md

```diff
@@ -153,6 +153,40 @@ For example, an image asset in the *wwwroot/images* folder is accessible to the
 
 [Multi-value cookies](xref:System.Web.HttpCookie.Values) aren't supported in ASP.NET Core. Create one cookie per value.
 
+## Partial app migration
+
+One approach is to create an IIS sub-application and only move certain routes from ASP.NET 4.x to ASP.NET Core while preserving the URL structure the app. For example, consider the relevant section from the *applicationHost.config* file:
```

Comment:
This seems to beg for more explanation. "One approach" to what? I also expect the flow of the thought to continue after the relevant applicationHost.config section, since it started with "Consider".  Something like "The following section from applicationhost.config file is an example of how partial app migration can be achieved by moving the /api route but leaving out the others... From IIS perspective, I don't see any problem.

Reply:
@bariscaglar any chance you could PR this and I'll close my PR?

Reply:
Sorry, I really can't. I am just here to give the IIS perspective since @shirhatti asked for it. After reading it from the point of view of a customer, I pointed out a couple of things that I found hard to follow in the documentation. I'll leave it to Sourabh to decide how to move on. From an IIS point of view I do not see any problem.